libr/anal/anal.c

```diff
@@ -55,6 +55,45 @@ static void zign_rename_for(void *user, int idx, const char *oname, const char *
 	r_sign_space_rename_for (anal, idx, oname, nname);
 }
 
+
+static void __anal_hint_on_bits(RAnal *a, ut64 addr, int bits, bool set) {
+	if (set) {
+		r_anal_build_range_on_hints (a, addr, bits);
+	}
+
+}
+
+
+static void __anal_hint_tree_calc_max_addr(RBNode *node) {
+	int i;
+	RAnalRange *range = container_of (node, RAnalRange, rb);
+	range->rb_max_addr = range->from;
+	for (i = 0; i < 2; i++) {
+		if (node->child[i]) {
+			RAnalRange *range1 = container_of (node->child[i],
+							   RAnalRange, rb);
+			if (range1->rb_max_addr > range->rb_max_addr) {
+				range->rb_max_addr = range1->rb_max_addr;
+			}
+		}
+	}
+}
+
+static int __anal_hint_range_tree_cmp(const void *a_, const RBNode *b_) {
+	const RAnalRange *a = (const RAnalRange *)a_;
```

Comment:
Does it need to cast void *? In above function you don't cast but you cast it here.

Reply:
fixed